### DISCUSSION TITLE
How to disable "select all" in tables.

### DISCUSSION THREAD

**Discussion by orzcc** [filamentphp/filament#6820] [Help] [answered]
My Records data is relatively large (over 100,000), and I don't want to calculate the total count when reading the table. So I used SimplePaginate, which actually doesn't need to be calculated anymore, but when debugging the code, I found that it still calculates when loading table data.

The call is displayed in filament/tables/src/Concerns/CanSelectRecords.php:87, which is this function: getAllSelectableTableRecordsCount.

But after careful inspection, it was found that there is no configuration to support turning off the "select all" option, and this feature is also hard-coded in Livewire.

**Comment by danharrin** [comment, marked as answer]
#4105, return `true` from `shouldSelectCurrentPageOnly()` on the Livewire component

**Reply by orzcc** [reply]
It works, thank you.
